packages/integration-tests/src/amplify_app_pool.ts

```diff
@@ -173,6 +178,13 @@ class DefaultAmplifyAppPool implements AmplifyAppPool {
       }
     }
   };
+
+  private retryableOperation = <T>(operation: () => Promise<T>) => {
+    return runWithRetry(operation, (error) => {
+      // Add specific error conditions here that warrant a retry
+      return error.message.includes('Unexpected token');
+    });
```

Comment:
```suggestion
    return runWithRetry(operation, (error) => {
      // Add specific error conditions here that warrant a retry
      return error.message.includes('Unexpected token') || error.message.includes('Bad control character');
    });
```

From recent run 
![image](https://github.com/user-attachments/assets/e0ab3cf5-1846-4f8c-a13e-7d3b3470a924)